app/src/main/res/values/strings.xml

```diff
@@ -353,24 +68,6 @@
     <!-- This label is shown above a list of apps that can be used to open a given link -->
     <string name="external_multiple_apps_matched_exit">Exit Private Browsing?</string>
```

Comment:
Are these two needed?

Reply:
Probably not: this is called by the `IntentUtils.handleExternalUri`, which we don't really support ATM. That being said, I haven't removed much code from existing methods that are called (e.g. how this method is probably called from MainActivity) – I just removed code that the IDE determined was unused. I figure we can clean these types of methods up incrementally, as we start changing the code that uses them.